src/components/Counter.js

```diff
@@ -20,10 +20,10 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => /* Fill me in */this.props.increment() }>
```

Comment:
i would suggest passing the function in directly instead of within a callback since functions can get expensive in memory and this anonymous functions isn't necessary :) 

```js
<button onClick={this.props.increment} >
```